ansible/aodn-database/roles/common/library/postgresql_exec.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/python
```

Comment:
This is just here for running arbitrary SQL for the functions. If replaced by something (e.g. liquibase, as above), could remove this module as well.